scripts/docs/extension_instructions.md

```diff
@@ -10,154 +10,91 @@ working with the same Bareflank workspace directory structure.
 
 ## Building and Integrating Extensions
 Extensions can be added to the Bareflank build system by using the
-*vmm_extension()* macro within a Bareflank build configuration file
-(bfconfig.cmake). The vmm_extension macro is compatible with all valid arguments
-to CMake's built-in
-[ExternalProject_Add()](https://cmake.org/cmake/help/v3.6/module/ExternalProject.html)
-function. For example, you could add the Bareflank extended apis extension from
-GitHub by adding the following to your build configuration file
-(bfconfig.cmake):
+`EXTENSION` variable to pass on the root directories of the extensions.
 
+In the top-level *CMakeLists.txt* file, you can setup and include extensions by
+using the `vmm_extension()` function. The vmm_extension function is a wrapper around
+the `add_subproject()` function, which makes it compatible with all valid arguments
+to CMake's built-in [ExternalProject_Add()](https://cmake.org/cmake/help/v3.6/module/ExternalProject.html)
```

Comment:
Not true anymore. add_dependency supports this, but add_subproject doesn't as it has a well defined API.